torch/export/_trace.py

```diff
@@ -96,6 +96,7 @@ class ExportDynamoConfig:
     reorderable_logging_functions: Set[Callable] = dataclasses.field(
         default_factory=set
     )
+    do_not_emit_runtime_asserts = True  # emit runtime asserts after AOTAutograd instead
 
```

Comment:
This isn't really necessary, and after https://github.com/pytorch/pytorch/pull/128599 isn't that much more efficient, but if we want to reason more about what guards/runtime asserts to emit, I guess this makes it a bit cleaner to do from the export side. Also I guess there's no point in running this twice.

Reply:
I think comments like this are better added to the code itself.